src/entities/WeeklyDebrief.ts

```diff
@@ -0,0 +1,42 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import { prop as Property, DocumentType } from '@typegoose/typegoose';
+import mongoose from 'mongoose';
+import { Article } from './Article';
+
+@ObjectType({ description: 'The Weekly Debrief Model' })
+export default class WeeklyDebrief {
+  @Field(() => ID)
+  id: string;
+
+  @Field()
+  @Property()
+  uuid: string; // uuid of user associated with this WeeklyDebrief
+
+  @Field()
+  @Property()
+  createdAt: Date;
+
+  @Field()
+  @Property()
+  expirationDate: Date;
+
+  @Field()
+  @Property({ default: 0 })
+  numShoutouts?: number;
+
+  @Field()
+  @Property({ default: 0 })
+  numBookmarkedArticles?: number;
+
+  @Field()
+  @Property({ default: 0 })
+  numReadArticles?: number;
+
+  @Field((type) => [Article])
+  @Property({ required: true, type: () => Article, default: [] })
+  readArticles: mongoose.Types.DocumentArray<DocumentType<Article>>;
```

Comment:
And then `readArticles: Article[]`

Reply:
I believe I've attempted this before and it didn't work. I think Kidus copied this from my implementation of followedPublications and `@Property({ required: true, type: () => Publication, default: [] })
  followedPublications: mongoose.Types.DocumentArray<DocumentType<Publication>>;` was the only way I could get a list of Publication objects after looking through several github issues. Not sure why though to be honest.